incubator/openvpn/values.yaml

```diff
@@ -48,6 +48,8 @@ persistence:
   size: 1Gi
 
 easyrsa:
+  clientCertExpire: "3650"
```

Comment:
Let's by default set client certificate to something shorter to indicate best practices. Maybe reduce to 30 days.